tests/spec/dirty_tests.py

```diff
@@ -50,7 +50,7 @@ class DirtyTest(object):
 \\S+ +\\d+ +\\d+ +\\d+ +\\d+ +\\d+ +\\S+ +\\S+ +\\S+ +\\S+ +{cmd}
 
 There are two ways you can fix this:
-  \\* temporarily: lsof -t playground/{service} | xargs kill -9
+  \\* temporarily: pgctl-fuser playground/{service} | xargs kill -9
   \\* permanently: http://pgctl.readthedocs.org/en/latest/user/quickstart.html#writing-playground-services
 
```

Comment:
In the case of --force, show this message, but instead of explaining the workarounds, say "Killing forcefully at user's request." or something similar.